### PR TITLE
Drop Travis CI

### PR DESCRIPTION
**Pull request by ihostage**

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -106,35 +102,3 @@ jobs:
         run: sbt ++$SCALA_VERSION 'testOnly -- xonly timefactor 5'
         env:
           SCALA_VERSION: ${{ matrix.scala }}
-
```
Comment (ihostage) [inline .github/workflows/build-test.yml:109]: Moved to  `2.1.x` branch

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -76,7 +73,6 @@ jobs:
         run: sbt doc
 
   tests:
-    if: ${{ github.base_ref != '2.1.x' }}
```
Comment (ihostage) [inline .github/workflows/build-test.yml:79]: Don't need now

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -4,9 +4,7 @@ on:
   push:
     branches: # Snapshots
       - main
-#      - 2.1.x # TODO: 2.1.x don't support sbt-ci-release yet
```
Comment (ihostage) [inline .github/workflows/publish.yml:7]: Don't need now

File: .mergify.yml [hunk 4]
```diff
@@ -7,45 +7,9 @@ queue_rules:
       - check-success=Check / Docs
       - check-success~=^Check / Tests \(Scala 2\.13\.(\d+) & JDK 11\)$
       - check-success~=^Check / Tests \(Scala 2\.13\.(\d+) & JDK 8\)$
-  - name: 2-1-x
```
Comment (ihostage) [inline .mergify.yml:10]: Second queue don't support in free version Mergify 😞

**Review by octonato — APPROVED** [review]
LGTM

**Pull request merged**